duckduckgo/Android
# Lhf/david/theming and UI consistency

Task/Issue URL: https://app.asana.com/0/1157893581871903/1200717485373414


**Description**:
Several components of the Android application are not following the desired guidelines defined in App Components
- Some icons in Toolbars
- Popup Menus (in lists, overflow menus)
- Options Menu created from the Toolbar
- Fire Button Bottom sheet (Icon and text color)
- Design of several list items (Unprotected Sites, Location Permissions)

To deal wit the last point, we've created two views `TwoLineListem` and 'SingleLineListItem' that are now used in all screens that display list. It's style has been updated to match the latest App Components in Figma. 

This PR also extracts the `DuckDuckGoActivity` into `common-ui`, in preparations for it to be used in other modules outside of `app`


**Steps to test this PR**:
1. Make sure branch compiles
2,. Open app, play around and see that nothing is out of place


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
